introduction/symfony1.rst

```diff
@@ -287,10 +287,10 @@ do the following:
         // app/config/routing.php
         use Symfony\Component\Routing\RouteCollection;
 
-        $collection = new RouteCollection();
-        $collection->addCollection($loader->import("@AcmeHelloBundle/Resources/config/routing.php"));
+        $routes = new RouteCollection();
```

Comment:
Some of the docs use `$routes` and some use `$collection` for `RouteCollection`, I thought that `$routes` sounded better but maybe `$routeCollection` is the best option? Any suggestion on this?

Reply:
I like `$routes` more than `$collection` and `$routeCollection`

Reply:
Although I like $routes more, in the examples below the calls to `$route->addCollection()` and `$route->setHost()` make more sense as `$routeCollection->addCollection()` and `$routeCollection->setHost()`.

Reply:
I disagree :)  `$routes->addCollection()` is easy to understand to me (just add a collection of routes to the existing routes) whereas `$routeCollection->addCollection()` is complicated (I'm replacing the existing collection? I'm merging two collections or creating a new collection? Is the result a route collection or a collection of route collections? etc.)